mscore/scoreview.cpp

```diff
@@ -2581,9 +2581,8 @@ void ScoreView::deselectAll()
 QVariant ScoreView::inputMethodQuery(Qt::InputMethodQuery query) const
       {
 //      qDebug("0x%x  %s", int(query), editData.element ? editData.element->name() : "-no element-");
-      // if editing a text object, place the InputMethod popup window just below the text
-      if (editData.dropElement && editData.dropElement->isTextBase()) {
-            TextBase* text = toTextBase(editData.dropElement);
+      if (editData.element && editData.element->isTextBase()) {
+            TextBase* text = toTextBase(editData.element);
```

Comment:
these two lines I believe are the crucial fix.  For some reason (I believe unintentionally) werner made a change:

https://github.com/musescore/MuseScore/commit/d2e452040b808825865efd4410a4a70cdb9edac7#diff-39acd419478f3f0d33e494c8b37942b0R2564

Which doesn't make sense to me, because the input method editor is not for text elements, not specifically drop text elements.